tests/test_custom_schema.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+
+from postgresql_audit import VersioningManager
+
+from .utils import last_activity
+
+
+@pytest.fixture()
+def cs_name():
+    return 'audit'
+
+
+@pytest.fixture()
+def cs_versioning_manager(cs_name):
+    return VersioningManager(schema_name=cs_name)
+
+
+@pytest.yield_fixture()
+def cs_activity_cls(base, cs_versioning_manager):
+    cs_versioning_manager.init(base)
+    yield cs_versioning_manager.activity_cls
+    cs_versioning_manager.remove_listeners()
+
+
+@pytest.yield_fixture()
+def cs_table_creator(base, connection, session, models, cs_versioning_manager,
+                     cs_name):
+    sa.orm.configure_mappers()
+    connection.execute('DROP SCHEMA IF EXISTS {} CASCADE'.format(cs_name))
+    tx = connection.begin()
+    cs_versioning_manager.activity_cls.__table__.create(connection)
+    base.metadata.create_all(connection)
+    tx.commit()
+    yield
+    base.metadata.drop_all(connection)
+    session.commit()
+
+
+@pytest.mark.usefixtures('cs_activity_cls',
+                         'cs_table_creator')
+class TestCustomSchemaActivityCreation(object):
+
+    def test_cs_insert(self, user, connection, cs_name):
```

Comment:
I think we could remove the cs prefix from these methods. It is implicitly clear from the class name.